xblockutils/public/tabs.css

```diff
@@ -0,0 +1 @@
+#xblock-editor{margin-bottom:0;height:100%}#xblock-editor .editor_content_and_tabs{width:100%;height:calc(100% - 55px)}#xblock-editor #editor_window_title{float:left}#xblock-editor input[type=number]{background-image:-moz-linear-gradient(#F2F2F2,#fff);background-image:-webkit-linear-gradient(#F2F2F2,#fff);background-image:linear,false,#F2F2F2,#fff;width:45%}#xblock-editor #editor_header{background-color:#e5e5e5;width:100%;top:0;height:42px}#xblock-editor #editor_header .editor_tabs{width:35%;min-width:190px;max-width:360px;float:right}#xblock-editor #editor_header .editor_tabs .editor_tab{float:right;padding:0;width:29%;margin:0 2%;height:35px;-webkit-border-radius:5px;-moz-border-radius:5px;-ms-border-radius:5px;-o-border-radius:5px;border-radius:5px;box-shadow:none;border:0;text-align:center}#xblock-editor #editor_header .editor_tabs .editor_tab a{padding:8px 0 0;width:100%;text-transform:uppercase;outline-color:transparent}#xblock-editor #editor_header .editor_tabs .ui-state-default.editor_tab{background:#e5e5e5}#xblock-editor #editor_header .editor_tabs .ui-state-default.editor_tab a{color:#424243}#xblock-editor #editor_header .editor_tabs .ui-state-active.editor_tab,#xblock-editor #editor_header .editor_tabs .ui-state-active.ui-state-default.editor_tab:hover{background:#626365}#xblock-editor #editor_header .editor_tabs .ui-state-active.editor_tab a,#xblock-editor #editor_header .editor_tabs .ui-state-active.ui-state-default.editor_tab:hover a{color:#fff}#xblock-editor #editor_header .editor_tabs .ui-state-default.editor_tab:hover{background-color:rgba(164,165,168,.75)}#xblock-editor .editor_content_wrapper{height:calc(100% - 42px);width:100%;-webkit-border-radius:3px;-moz-border-radius:3px;-ms-border-radius:3px;-o-border-radius:3px;border-radius:3px;background-color:#fff;overflow-y:scroll;bottom:0;z-index:11;transition:height 1s ease-in-out 0;-webkit-transition:height 1s ease-in-out 0;-moz-transition:height 1s ease-in-out 0;border:none}#xblock-editor label{padding-right:10px}.wrapper-comp-settings label input{font-size:11px;float:right;width:70%}#xblock-editor .wrapper-comp-settings .list-input.settings-list .field.comp-setting-entry select{padding:5px}
```

Comment:
This file looks generated. Wouldn't it make sense to include the source code instead? In the current minified form, this looks difficult to edit.